node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/LocksResponse.java

```diff
@@ -0,0 +1,73 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.provision.restapi;
+
+import com.yahoo.container.jdisc.HttpResponse;
+import com.yahoo.slime.Cursor;
+import com.yahoo.slime.JsonFormat;
+import com.yahoo.slime.Slime;
+import com.yahoo.vespa.curator.LockInfo;
+import com.yahoo.vespa.curator.ThreadLockInfo;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.time.Instant;
+import java.util.List;
+
+public class LocksResponse extends HttpResponse {
```

Comment:
Missing author/javadoc. Applies to some of the other classes as well.

Reply:
Fixed